docs/reference/tab-widgets/inference-api/infer-api-ingest-pipeline.asciidoc

```diff
@@ -18,7 +18,7 @@ PUT _ingest/pipeline/cohere_embeddings
   "processors": [
     {
       "inference": {
-        "model_id": "cohere_embeddings", <1>
+        "inference_id": "cohere_embeddings", <1>
```

Comment:
Sorry my response was confusing. What I meant was that we refer to this field as the `inference_id` when actually performing the put request for the inference configuration. But the it is actually `model_id` in the ingest processor. So let's change this back to `model_id`.